plugins/ConditionalRestrictions.py

```diff
@@ -128,16 +141,42 @@ def way(self, data, tags, nds):
           err.append({"class": 33501, "subclass": 2 + stablehash64(tag + '|' + tag_value), "text": T_("Mismatch in the number of parentheses in \"{0}\"", tag)})
           continue
 
-      # Check the position of AND is ok
       if not bad_tag:
         for condition in conditions:
-          tmp_cond = " " + condition.replace(" ", "  ") + " "
-          tmp_ANDsplitted = tmp_cond.upper().split(" AND ")
-          for splittedANDpart in tmp_ANDsplitted:
-            if len(splittedANDpart.strip()) == 0:
-              err.append({"class": 33501, "subclass": 4 + stablehash64(tag + '|' + tag_value), "text": T_("Missing condition before or after AND combinator in \"{0}\"", tag)})
-              bad_tag = True
-              break
+          condition_ANDsplitted = list(map(str.strip, self.ReAND.split(condition)))
+          # Check the position of AND is ok
+          if "" in condition_ANDsplitted:
+            err.append({"class": 33501, "subclass": 4 + stablehash64(tag + '|' + tag_value), "text": T_("Missing condition before or after AND combinator in \"{0}\"", tag)})
+            bad_tag = True
+            break
```

Comment:
Why break and not continue. Other conditions can still be checked, cannot ?

Reply:
True. I'd prefer throwing a single "invalid" warning where it's easy to do; one would only see one marker on the front-end anyway for a single object, and the errors may be related to a single typo. Would you like me to adjust it?
(In that case I'll also have to add `condition` to the hash as multiple conditions might have the same error)

Reply:
Yes I prefer. The Osmose frontend map is not the only to show reported issues.

Reply:
Will do (later today)

Reply:
Implemented in #1431